### PR TITLE
Standalone Arrow Client

### PR DESCRIPTION
**Pull request by DarthMax**

### CODE REVIEW

**Comment by FlorentinD** [comment]
[Include neo4j_arrow into the GDS Python Client](https://trello.com/c/O3akLUXj/9958-include-neo4jarrow-into-the-gds-python-client)

File: graphdatascience/query_runner/gds_arrow_client.py [hunk 1]
```diff
@@ -77,25 +94,508 @@ def __init__(
...
+        list_node_labels : bool
+            A flag that indicates whether the node labels should be included in the result
+        concurrency : Optional[int]
+            The number of threads used on the server side when serving the data
+
+        Returns
+        -------
+        DataFrame
+            The requested node property as a DataFrame
+        """
```
Comment (soerenreichardt) [inline graphdatascience/query_runner/gds_arrow_client.py:162]: Nice method docs

File: graphdatascience/query_runner/gds_arrow_client.py [hunk 2]
```diff
@@ -77,25 +94,508 @@ def __init__(
...
+        The import process accepts triplets as input data.
+
+        Parameters
+        ----------
+        database: str
+            The name used to identify the database and the import process
+        id_type : Optional[str]
+            Sets the node id type used in the input data. Can be either `INTEGER` or `STRING` (default is `INTEGER`)
+        id_property : Optional[str]
+            The node property key which stores the node id of the input data (default is `originalId`)
```
Comment (soerenreichardt) [inline graphdatascience/query_runner/gds_arrow_client.py:375]: Where are these default values set?

**Review by soerenreichardt — APPROVED** [review]
Very nice stuff!

File: graphdatascience/query_runner/gds_arrow_client.py (hunk 2 above)
Comment (DarthMax) [reply graphdatascience/query_runner/gds_arrow_client.py:375]: In GDS, the values are optional in the config and have defaults on the GDS side.

**Pull request merged**